luarules/gadgets/game_custom_map_tidal.lua

```diff
@@ -1,25 +1,27 @@
+if not gadgetHandler:IsSyncedCode() then return false end
+
 function gadget:GetInfo()
 	return {
 		name = "Custom Map Tidal",
 		desc = "Sets map tidal for modoption via engine call",
-		author = "",
-		date = "",
-		license = "Horses",
+		author = "robert the pie",
+		date = "December 2023",
+		license = "GPLv2 or late",
 		layer = 0,
 		enabled = true  --  loaded by default?
 	}
 end
 
-if gadgetHandler:IsSyncedCode() then
-
 function gadget:Initialize()
 	local newTidal = Spring.GetModOptions().map_tidal
-	-- set tidal doesn't u
-	local tidalSpeeds = {high=23,medium=18,low=13,off=0}
+	local tidalSpeeds = {
+		high=23,
+		medium=18,
+		low=13,
+		unchanged=nil,
```

Comment:
you lost a 0

Reply:
i'd need to update the mod options file for that and i deemed it would probably be better to just turn the tidal unit off instead if someone wanted it to be zero.